modules/charon-core/src/main/java/org/wso2/charon3/core/protocol/endpoints/GroupResourceManager.java

```diff
@@ -370,14 +371,14 @@ public SCIMResponse listWithGET(UserManager userManager, String filter, Integer
 
             // API group should pass a user manager to GroupResourceEndpoint.
             if (userManager != null) {
-                List<Object> tempList = userManager
-                        .listGroupsWithGET(rootNode, startIndex, count, sortBy, sortOrder, domainName,
-                                requiredAttributes);
+                List<Object> tempList = userManager.listGroupsWithGET(rootNode, startIndex, count,
+                        sortBy, sortOrder, domainName, requiredAttributes);
                 return processGroupList(tempList, encoder, attributes, excludeAttributes, startIndex);
             } else {
                 String error = "Provided user manager handler is null.";
-                // Log the error as well.
-                // Throw internal server error.
+                if(logger.isDebugEnabled()){
```

Comment:
Formatting issue. Checkstyle is failing.

Reply:
Fixed: 706d6b68a3568dfb5ced373c02ae61022df156ad